lib/workers/repository/process/vulnerabilities.ts

```diff
@@ -91,51 +97,367 @@ export class Vulnerabilities {
+        parsedCvss.cvss3OverallSeverityText.slice(1);
+
+      return [parsedCvss.baseScore.toFixed(1), severityLevel];
+    } catch (err) {
+      logger.debug(`Error processing CVSS vector ${vector}`);
+    }
+
+    return ['0', ''];
+  }
+
+  private static generatePrBodyNotes(
+    vulnerability: Osv.Vulnerability
+  ): string[] {
+    let aliases = [vulnerability.id].concat(vulnerability.aliases ?? []).sort();
+    aliases = aliases.map((id) => {
+      if (id.startsWith('CVE-')) {
+        return `[${id}](https://nvd.nist.gov/vuln/detail/${id})`;
+      } else if (id.startsWith('GHSA-')) {
+        return `[${id}](https://github.com/advisories/${id})`;
+      } else if (id.startsWith('GO-')) {
+        return `[${id}](https://pkg.go.dev/vuln/${id})`;
+      } else if (id.startsWith('RUSTSEC-')) {
+        return `[${id}](https://rustsec.org/advisories/${id}.html)`;
+      }
+
+      return id;
+    });
+
+    let content = '\n\n---\n\n### ';
+    content += vulnerability.summary ? `${vulnerability.summary}\n` : '';
+    content += `${aliases.join(' / ')}\n`;
+    content += `\n<details>\n<summary>More information</summary>\n`;
+    content += `### Details\n${vulnerability.details ?? 'No details'}\n`;
+    content += '### Severity\n';
+    if (vulnerability.severity?.[0].score) {
+      const [score, severity] = this.evaluateCvssVector(
+        vulnerability.severity[0].score
       );
+      content += `- Score: ${score} / 10 (${severity})\n`;
+      content += `- Vector: \`${vulnerability.severity?.[0].score}\`\n`;
+    } else {
+      content += 'Unknown severity\n';
+    }
+    content += `\n### References\n${
+      vulnerability.references
+        ?.map((ref) => {
+          return `- [${ref.url}](${ref.url})`;
+        })
+        .join('\n') ?? 'No references'
+    }`;
+
+    let attribution = '';
+    if (vulnerability.id.startsWith('GHSA-')) {
+      attribution = ` and the [GitHub Advisory Database](https://github.com/github/advisory-database) ([CC-BY 4.0](https://github.com/github/advisory-database/blob/main/LICENSE.md))`;
+    } else if (vulnerability.id.startsWith('GO-')) {
+      attribution = ` and the [Go Vulnerability Database](https://github.com/golang/vulndb) ([CC-BY 4.0](https://github.com/golang/vulndb#license))`;
+    } else if (vulnerability.id.startsWith('PYSEC-')) {
+      attribution = ` and the [PyPI Advisory Database](https://github.com/pypa/advisory-database) ([CC-BY 4.0](https://github.com/pypa/advisory-database/blob/main/LICENSE))`;
+    } else if (vulnerability.id.startsWith('RUSTSEC-')) {
+      attribution = ` and the [Rust Advisory Database](https://github.com/RustSec/advisory-db) ([CC0 1.0](https://github.com/rustsec/advisory-db/blob/main/LICENSE.txt))`;
+    }
+    content += `\n\nThis data is provided by [OSV](https://osv.dev/vulnerability/${vulnerability.id})${attribution}.\n`;
+    content += `</details>`;
+
+    return [sanitizeMarkdown(content)];
```

Comment:
are you sure we need to sanitize here? i think this is already done before posting to platform

Reply:
I think it's needed here because `sanitizeMarkdown()` is just applied on release notes, not on everything that's posted:

https://github.com/renovatebot/renovate/blob/8438e3068584ef2b5bda7bc994d0a53b3cd548fd/lib/workers/repository/update/pr/body/changelogs.ts#L37

The only other spot where it's called besides release notes, is the description text for Github vuln alerts:
https://github.com/renovatebot/renovate/blob/8438e3068584ef2b5bda7bc994d0a53b3cd548fd/lib/workers/repository/init/vulnerability.ts#L199